.travis.yml

```diff
@@ -1,42 +1,43 @@
 sudo: required
-dist: trusty
+dist: xenial
 
-# install the pre-release chef-dk.  Use chef-stable-precise to install the stable release
+# Install the pre-release chef tools
 addons:
   apt:
     sources:
-      - chef-current-trusty
+      - chef-current-xenial
     packages:
-      - chefdk
```

Comment:
Is this value correct? 
https://github.com/travis-ci/travis-cookbooks/blob/master/.travis.yml